src/apis/tasks.js

```diff
@@ -0,0 +1,113 @@
+import { getToken } from "../utils/utils";
+
+async function getTasks(apiEndPoint) {
+  const response = await fetch(
+    `${process.env.REACT_APP_SERVER_DOMAIN}/api/v1/tasks${apiEndPoint}`,
+    {
+      headers: {
+        token: getToken(),
+      },
+    }
+  );
+  const responseData = await response.json();
+  if (!response.ok) {
+    const error = new Error();
+    error.message = responseData.message;
+    error.status = response.status;
+    error.errorMessage = responseData.errorMessage;
+    throw error;
+  }
+  return responseData;
+}
+
+async function createTask({ body }) {
+  const response = await fetch(
+    `${process.env.REACT_APP_SERVER_DOMAIN}/api/v1/tasks/create`,
+    {
+      method: "POST",
+      body: JSON.stringify(body),
+      headers: {
+        "Content-Type": "application/json",
+        token: getToken(),
+      },
+    }
+  );
+  const responseData = await response.json();
+  if (!response.ok) {
+    const error = new Error();
+    error.message = responseData.message;
+    error.status = response.status;
+    error.errorMessage = responseData.errorMessage;
+    throw error;
+  }
```

Comment:
i can see repeatation of same code multiple times, you can seperate it out in single function to reduce LOC.